client/components/section-nav/style.scss

```diff
@@ -347,6 +347,17 @@
 	}
 }
 
+.section-nav-tab__title {
```

Comment:
My understanding is that this CSS should live in `media-library/title-item.css` and be named something like `.media-library__title`.